crt_portal/cts_forms/forms.py

```diff
@@ -1,86 +1,65 @@
-    LOCATION_QUESTIONS,
-    ELECTION_QUESTION,
-    WORKPLACE_QUESTIONS,
-    PUBLIC_QUESTION,
-    POLICE_QUESTIONS,
-    EDUCATION_QUESTION,
-    PROTECTED_CLASS_QUESTION,
-    DATE_QUESTIONS,
-    SUMMARY_QUESTION,
-    SUMMARY_HELPTEXT,
-)
-
+from .model_variables import (COMMERCIAL_OR_PUBLIC_ERROR,
+                              COMMERCIAL_OR_PUBLIC_PLACE_CHOICES,
+                              COMMERCIAL_OR_PUBLIC_PLACE_HELP_TEXT,
+                              CORRECTIONAL_FACILITY_LOCATION_CHOICES,
+                              CORRECTIONAL_FACILITY_LOCATION_TYPE_CHOICES,
+                              DATE_ERRORS, DISTRICT_CHOICES, ELECTION_CHOICES,
+                              EMPLOYER_SIZE_CHOICES, EMPLOYER_SIZE_ERROR,
+                              EMPTY_CHOICE, INCIDENT_DATE_HELPTEXT,
+                              POLICE_LOCATION_ERRORS,
+                              PRIMARY_COMPLAINT_CHOICES,
+                              PRIMARY_COMPLAINT_CHOICES_TO_EXAMPLES,
+                              PRIMARY_COMPLAINT_CHOICES_TO_HELPTEXT,
+                              PRIMARY_COMPLAINT_ERROR, PROTECTED_CLASS_CHOICES,
+                              PROTECTED_CLASS_ERROR,
+                              PUBLIC_OR_PRIVATE_EMPLOYER_CHOICES,
+                              PUBLIC_OR_PRIVATE_EMPLOYER_ERROR,
+                              PUBLIC_OR_PRIVATE_SCHOOL_CHOICES,
+                              SECTION_CHOICES, SERVICEMEMBER_CHOICES,
+                              SERVICEMEMBER_ERROR, STATES_AND_TERRITORIES,
+                              STATUS_CHOICES, STATUTE_CHOICES,
+                              VIOLATION_SUMMARY_ERROR, VOTING_ERROR,
+                              WHERE_ERRORS)
+from .models import (CommentAndSummary, HateCrimesandTrafficking,
+                     ProtectedClass, Report)
 from .phone_regex import phone_validation_regex
-
-import logging
+from .question_group import QuestionGroup
+from .question_text import (CONTACT_QUESTIONS, DATE_QUESTIONS,
+                            EDUCATION_QUESTION, ELECTION_QUESTION,
+                            HATECRIME_QUESTION, HATECRIME_TITLE,
+                            LOCATION_QUESTIONS, POLICE_QUESTIONS,
+                            PRIMARY_REASON_QUESTION, PROTECTED_CLASS_QUESTION,
+                            PUBLIC_QUESTION, SERVICEMEMBER_QUESTION,
+                            SUMMARY_HELPTEXT, SUMMARY_QUESTION,
+                            WORKPLACE_QUESTIONS)
+from .widgets import (ComplaintSelect, CrtMultiSelect,
+                      CrtPrimaryIssueRadioGroup, UsaCheckboxSelectMultiple,
+                      UsaRadioSelect)
 
 logger = logging.getLogger(__name__)
+User = get_user_model()
 
 
 def _add_empty_choice(choices):
     """Add an empty option to list of choices"""
     return (EMPTY_CHOICE,) + choices
 
 
+class UserChoiceField(ModelChoiceField):
+    def label_from_instance(self, obj):
+        return f"{obj.last_name}, {obj.first_name}"
```

Comment:
I think we need the search to be by username. 

We don't have first name or last name populated in production. In theory, we could add people manually, but I still think that would be problematic. If we could get the claims working, we could use this feature, but that isn't a near term solution because we would have to coordinate debugging with 2 other DOJ teams.

Reply:
tackled in 3ffb71854c377ab2bad64b8031a613666fe0dcad